### PR TITLE
Added complementary classes

### PR DESCRIPTION
**Pull request by samirbr**
#### Changes proposed in this pull request:

* Added complementary classes for IntelliSense completion.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @samirbr! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/core/src/common/classes.ts [hunk 1]
```diff
@@ -64,15 +71,31 @@ export const ELEVATION_4 = "pt-elevation-4";
...
+export const CHECKBOX = "pt-checkbox";
+export const RADIO = "pt-radio";
+export const SWITCH = "pt-switch";
+export const FILE_UPLOAD = "pt-file-upload";
+export const FILE_UPLOAD_INPUT = "pt-file-upload-input";
+
+export const INTENT_PRIMARY = "pt-intent-primary";
+export const INTENT_SUCCESS = "pt-intent-success";
+export const INTENT_WARNING = "pt-intent-warning";
+export const INTENT_DANGER = "pt-intent-danger";
```
Comment (giladgray) [inline packages/core/src/common/classes.ts]: hmm we already have the `Intent` enum used by components that support intent modifiers and `Classes.intentClass()` to convert the enum to a class name. 

i do still recognize the simplicity and consistency of also supplying these constants.

File: packages/core/src/common/classes.ts [hunk 2]
```diff
@@ -116,6 +144,11 @@ export const TAB_LIST = "pt-tab-list";
 export const TAB_PANEL = "pt-tab-panel";
 export const TABS = "pt-tabs";
 
+export const TABLE = "pt-table";
+export const CONDENSED = "pt-condensed";
+export const STRIPED = "pt-striped";
+export const BORDERED = "pt-bordered";
```
Comment (giladgray) [inline packages/core/src/common/classes.ts]: interesting quirk: these last three are technically modifiers but they're only supported by this one component. 

I wonder if we should call the constants `TABLE_*` to indicate that's where they're intended to be used? 

And then if we should change the classes to `pt-table-*` in 2.x?

Comment (giladgray) [inline packages/core/src/common/classes.ts]: same goes for `NO_STRIPES` and `NO_ANIMATION` above.

Comment (llorca) [inline packages/core/src/common/classes.ts]: Makes sense, they're not generic modifiers indeed 👍

Comment (giladgray) [inline packages/core/src/common/classes.ts]: ok @samirbr please make this change

File: packages/core/src/common/classes.ts (hunk 1 above)
Comment (llorca) [inline packages/core/src/common/classes.ts]: We should probably moves these intent variables up in the modifiers section, right?

**Comment by giladgray** [comment]
🎩  thank you @samirbr!

**Pull request merged**